.github/workflows/ci.yaml

```diff
@@ -49,6 +49,7 @@ jobs:
       - name: Install pycoast
         shell: bash -l {0}
         run: |
+          python -m pip install pytest-lazy-fixtures
```

Comment:
Is this because it isn't available on conda-forge? Maybe we can get that package added.

Reply:
Oh and actually you can list pip dependencies inside a conda environment.yaml. That might be "cleaner" in that all dependencies are in one spot.

Reply:
Yes, you are right.
Bu I see that you already merged this PR.
Can I just push the new change to the same branch or do I need to open a new PR?

Reply:
Don't worry about it. It's fine like this probably. If it really bothers you i think you need to create a new branch and new PR as your old branch is no longer synced with main.

Reply:
OK, thanks